### PR TITLE
[WIP] Actual Budget Vision 

### PR DESCRIPTION
**Pull request by MatissJanis**
Original PR: https://github.com/actualbudget/docs/pull/175

### CODE REVIEW

File: docs/vision.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+This document provides an outline of the vision of the open source version of Actual Budget, where the project would like to be, what the goals are and some of the history around Actual Budget. 
+
+### What Makes Actual Budget Great
+
+Some of the things that make Actual Budget stand out from the rest include;
+
+* Local-first
+
+Actual is local first meaning that the database that powers Actual Budget and ultimatley your budget live on your device, we don't store it on a server, now your wondering what actual server is for, well that allows us to sync changes between budgets, to do this we only send the changes to the server, it stores them and the client pulls down the change(s)
```
Comment (shall0pass) [inline docs/vision.md:13]:
```suggestion
Actual is local first meaning that the database that powers Actual Budget and ultimatley your budget live on your device. The Actual Budget project doesn't store your information on a centeral server. Now your wondering what actual server is for, well that allows you to sync changes between your budgets from your personal server install. To do this, only changes are sent to the server and ared stored. Your client(s) pull down the change(s)
```

File: docs/vision.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+Instead, Actual Budget will focus on developing an excellent web experience by combining the underlying technology and some UI components from the desktop version with some components maintained separately for mobile and desktop where there need to be significant differences in presentation. For more details, [see our dedicated mobile strategy issue](https://github.com/actualbudget/actual/issues/804).
+
+### The Future
+
+Some of the high-level features we would like to add to Actual Budget in the future include:
+
+* Goal templates moved to the UI.
+* Mobile transaction entry.
+* Implementing and testing syncing providers other than Nordigen.
```
Comment (shall0pass) [inline docs/vision.md:101]:
```suggestion
* Implementing and testing syncing providers in addition to GoCardless.
```

File: docs/vision.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+* Enabling support for Nordigen and other syncing providers by default.
```
Comment (shall0pass) [inline docs/vision.md:102]:
```suggestion
* Enabling support for additional syncing providers by default.
```

File: docs/vision.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+Some of the things that make Actual Budget stand out from the rest include;
+
+* Local-first
+
+Actual is local first meaning that the database that powers Actual Budget and ultimatley your budget live on your device, we don't store it on a server, now your wondering what actual server is for, well that allows us to sync changes between budgets, to do this we only send the changes to the server, it stores them and the client pulls down the change(s)
+
+* Open 
+
+Actual is Open Source, the code is open for anyone to see, view and contribute to. You own your data, you can do whatever you want with it.
```
Comment (shall0pass) [inline docs/vision.md:17]:
```suggestion
Actual is Open Source utilizing the MIT license, the code is open for anyone to see, view and contribute to. You own your data, you can do whatever you want with it.
```

File: docs/vision.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+Some of the high-level features we would like to add to Actual Budget in the future include:
+
+* Goal templates moved to the UI.
+* Mobile transaction entry.
+* Implementing and testing syncing providers other than Nordigen.
+* Enabling support for Nordigen and other syncing providers by default.
+* Darkmode.
+* custom reports.
+* and more
```
Comment (shall0pass) [inline docs/vision.md:105]: ## Third party use
Actual is MIT licensed which allows for use of the software without restriction. The Actual project and community appreciate any submissions from 3rd party contributors to enhance and improve our amazing project. This appreciation extends to any single volunteer or company participants alike.  Let's continue to make Actual Budget the best personal finance tool together!

**Review by MatissJanis — COMMENTED** [review]
Can't approve since I opened this PR, but thanks for writing all this up @shall0pass ! I 100% agree with what you've noted down here.

**Review by youngcw — APPROVED** [review]

**Comment by shaankhosla** [comment]
This looks great!

File: docs/vision.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+### Local-first
+
+Actual is local first, meaning that the database that powers Actual Budget, and ultimately your budget, lives on your device. We don't store it on a server. Now you're wondering, what 'actual server' is for? Well, that allows you to sync changes to your budget on different devices. To do this, only the changes to the budget are sent to the server. The server stores the messages, and the client pulls down the change(s).
+
+### Open 
+
+Actual is Open Source utilizing the MIT license. The code is open for anyone to see, view, and contribute. You own your data; you can do whatever you want with it.
+
+### User Interface
+
```
Comment (trevdor) [inline docs/vision.md:31]: Please don’t interpret this as bringing up the recent “unlock” discussion :) , but I think a note here would be nice along the lines that we make mistakes easy to undo. Similar to the early Macintosh philosophy.

File: docs/vision.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
+---
+title: Actual Budget Vision
+---
+
+**Vision statement**: Our vision is to be the leading OSS personal finance software used by everyone in any stage of their financial journey.
```
Comment (trevdor) [inline docs/vision.md:5]: Suggestion: drop “our vision is” and let the heading speak for itself

**Review by youngcw — APPROVED** [review]

**Review by carkom — APPROVED** [review]
LGTM!

**Comment by MatissJanis** [comment]
@shall0pass are you happy to merge this?

**Comment by shall0pass** [comment]
I'm happy if you're all happy 😊. I didn't want to give the final approval since I authored a fair bit of it.

**Review by shall0pass — APPROVED** [review]

**Comment by MatissJanis** [comment]
LGTM :+1:

**Pull request merged**